static/absolute/js/participants.js

```diff
@@ -1,10 +1,31 @@
+const socket = io();
+socket.emit("join", getEventId());
+
+socket.on("refresh", () => {
+  console.log(
+    "Refresh message received from server. Updating participants page."
+  );
+  showParticipants();
```

Comment:
a new member, etc, could also change search results, right? does this need to refresh search as well?

Reply:
Actually, I see that's handled below... should those two be joined?

Reply:
I'm not sure I understand what you mean. 

The refresh message emitted from the server is sent to members of a session whether they are looking at the search page or the participants page. In the participants page, the only thing that needs to be updated is the list of participants.